src/main/msp/msp.c

```diff
@@ -343,16 +343,14 @@ static void mspFcSetPassthroughCommand(sbuf_t *dst, sbuf_t *src, mspPostProcessF
     }
 }
 
-// TODO: Remove the pragma once this is called from unconditional code
-#pragma GCC diagnostic ignored "-Wunused-function"
-static void configRebootUpdateCheckU8(uint8_t *parm, uint8_t value)
+// TODO: Remove the MAYBE_UNUSED once this is called from unconditional code
```

Comment:
Not sure about comment - `MAYBE_UNUSED` is possibly enough

Reply:
Okay, removed comment